yt/data_objects/tests/test_sph_data_objects.py

```diff
@@ -170,7 +170,7 @@ def test_ray():
 def test_cutting():
     ds = fake_sph_orientation_ds()
     for (normal, center), answer in CUTTING_ANSWERS.items():
-        for i in range(-1, 2):
+        for _ in range(3):
```

Comment:
seems like another broken test. @matthewturk, any idea what the intention here was ?

Reply:
So `range(-1,2)` gives `[-1, 0, 1]`.  My guess is that this is supposed to be an offset.  0.1 is within the radius of the smoothing length for all the particles, so what the test is probably trying to do is to offset the center along the axis.  I'd guess that maybe either `c + 0.1 * c` is supposed to be `c + 0.1 * i` or `cen[i]` gets offset or something was the original intention.

Reply:
sounds likely. I'll update this, but then I suppose it is excepted that the test will fail since it was apparently validated with wrong data.

Reply:
welp. It didn't fail. https://trello-attachments.s3.amazonaws.com/58e3f7c543422d7f3ad84f33/5f0751415f713e68c3cd2653/41a92bd352d8a14b62152d3c79f759df/coverage.png

Reply:
Good news!

Reply:
Is it ? I'm now suspecting the test isn't really doing anything

Reply:
I think the simpler answer is that it's passing.  What it's doing is checking that when we select a cutting plane that we are also selecting the particles whose smoothing length puts them within the range of the cutting plane.

If we were to reduce the smoothing lengths to, so, 1e-6, it would potentially fail.  Which would mean it is indeed working.